jupyter_server/services/contents/handlers.py

```diff
@@ -47,21 +51,30 @@ def validate_model(model, expect_content):
             f"Missing Model Keys: {missing}",
         )
 
-    maybe_none_keys = ["content", "format"]
-    if expect_content:
-        errors = [key for key in maybe_none_keys if model[key] is None]
-        if errors:
-            raise web.HTTPError(
-                500,
-                f"Keys unexpectedly None: {errors}",
-            )
-    else:
-        errors = {key: model[key] for key in maybe_none_keys if model[key] is not None}  # type: ignore[assignment]
-        if errors:
-            raise web.HTTPError(
-                500,
-                f"Keys unexpectedly not None: {errors}",
-            )
+    def validate_in_or_not(statement: bool, model: Dict[str, Any], maybe_none_keys: List[str]):
```

Comment:
Can you move this to a module-level private function please?

Reply:
Thanks for responding so quickly. I have made the changes.